app/assets/javascripts/dashboard/views/spotlight_view.js

```diff
@@ -0,0 +1,157 @@
+// flow
+import * as React from "react";
+import { transformDatasets } from "dashboard/views/dataset_view";
+import GalleryDatasetView from "dashboard/views/gallery_dataset_view";
+import Request from "libs/request";
+import { Spin, Layout, Row, Col } from "antd";
+import type { DatasetType } from "dashboard/views/dataset_view";
+
+const { Header, Content, Footer } = Layout;
+
+type State = {
+  datasets: Array<DatasetType>,
+  isLoading: false,
+};
+
+class SpotlightView extends React.PureComponent<{}, State> {
+  state = {
+    datasets: [],
+    isLoading: true,
+  };
+
+  componentDidMount() {
+    this.fetchData();
+  }
+
+  async fetchData(): Promise<void> {
+    const url = "/api/datasets";
+    this.setState({ isLoading: true });
+    const datasets = await Request.receiveJSON(url);
+
+    const transformedDatasets = transformDatasets(datasets);
```

Comment:
In the long run, we probably should have a module which takes care of these transformations. Then, we could call something like `API.fetchDatasets()` which would internally use receiveJSON + the appropriate URL and then transform the data. Similarly, to the old backbone model, but a bit more transparent.